app_dart/lib/src/model/ci_yaml/ci_yaml.dart

```diff
@@ -83,6 +84,11 @@ class CiYaml {
     return enabledTargets;
   }
 
+  /// Gets the first [Target] matching [builderName] or null.
+  Target? getFirstPostsubmitTarget(String builderName) {
```

Comment:
Can `getIgnoreFlakiness` also be updated to use this?

Reply:
done